controllers/glanceapi_controller.go

```diff
@@ -671,11 +689,12 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
 		},
 	}
 	err = configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
+
 	if err != nil {
-		return nil
+		errs = append(errs, err)
 	}
 
-	return nil
+	return k8serrors.NewAggregate(errs)
```

Comment:
I think these can be reversed too.  I will look into the operator not logging the individual errors that you mentioned.